src/iop/lowpass.c

```diff
@@ -142,55 +155,74 @@ int groups()
 int legacy_params(dt_iop_module_t *self, const void *const old_params, const int old_version,
                   void *new_params, const int new_version)
 {
-  if(old_version == 1 && new_version == 4)
+  if(old_version == 1 && new_version == 5)
   {
     const dt_iop_lowpass_params1_t *old = old_params;
     dt_iop_lowpass_params_t *new = new_params;
     new->order = old->order;
     new->radius = fabs(old->radius);
     new->contrast = old->contrast;
+    new->cfulcrum = 50.0f;
```

Comment:
default cfulcrum is 50.

Reply:
> default cfulcrum is 50.

Yes. Can you explicit a bit more what you mean ?